### DISCUSSION TITLE
Option to enable s3 and s3 compatible bucket storage in deployment?

### DISCUSSION THREAD

**Discussion by marrenia** [ubergeek77/Lemmy-Easy-Deploy#73] [Ideas]
As far as I am aware the current version of pictrs that lemmy uses allows configuration of an s3 bucket system like amazons s3 or backblaze b2 - however as far as i am aware editing the lemmy config files while using lemmy easy deploy leads to issues...would it be possible to add an option to the config.env and deploy script to add an s3 bucket block to the config files so we dont have to edit the files manually and potentially mess things up?

**Comment by ubergeek77** [comment]
Hey there!

Sorry to disappoint, but I won't be adding this feature to this project. I consider that to be an advanced use case, requiring enough knowledge of AWS to set that up, configure credentials, do it all in a way that keeps the S3 bucket secure and not overbilling for resources - it is simply not for beginners.

If you like the convenience of this script, but want these extra features, I do give you enough control over this deployment to do basically whatever you like:

https://github.com/ubergeek77/Lemmy-Easy-Deploy/blob/main/ADVANCED_CONFIGURATION.md

However, it is up to the end user to figure out the best approach and to implement it accordingly.

Sorry!
